packages/mpv/build.sh

```diff
@@ -4,10 +4,12 @@ TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
 # Update both mpv and mpv-x to the same version in one PR.
 TERMUX_PKG_VERSION="0.38.0"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/mpv-player/mpv/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=86d9ef40b6058732f67b46d0bbda24a074fae860b3eaae05bab3145041303066
 TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_DEPENDS="ffmpeg, libandroid-glob, libandroid-support, libarchive, libass, libcaca, libiconv, liblua52, libsixel, libuchardet, openal-soft, pulseaudio, rubberband, zlib, libplacebo"
+TERMUX_PKG_DEPENDS="alsa-lib, ffmpeg, jack, libandroid-glob, libandroid-support, libarchive, libass, libcaca, libiconv, liblua52, libsixel, libuchardet, openal-soft, pulseaudio, rubberband, zlib, libplacebo"
+TERMUX_PKG_BUILD_DPENEDS="jack2"
```

Comment:
Typo, DPENEDS -> DEPENDS. 
It works anyways at the moment since package depends on jack which depends on jack2, I don't think there is any reason to add jack2 to BUILD_DEPENDS

Reply:
Thanks for pointing out! And I removed that and just force-pushed. 

I thought when the metapackage `jack` indicates that (which I took reference to `vulkan-loader`): 
```
TERMUX_PKG_DEPENDS="jack2"
TERMUX_PKG_ANTI_BUILD_DEPENDS="jack2"
```
, then in build time `mpv` will not have `jack2` available. 

Instead, it seems that the first candidate of metapackage is installed during build time, right?

Reply:
It is still so puzzling to me why `mpv` can detect jack2 backend in build time (the built `mpv --ao=help` shows jack). I am asking because `ardour` cannot detect jack backend when `jack2` is not on the `TERMUX_PKG_BUILD_DEPENDS` (https://github.com/knyipab/termux-packages/actions/runs/9529197300). 
And FYR that `dev/pipewire-group` branch has a slightly different `jack` metapackage: 
```
TERMUX_PKG_DEPENDS="pipewire-jack | jack2"
TERMUX_PKG_ANTI_BUILD_DEPENDS="pipewire-jack, jack2"
```

Would that be because `mpv` and `jack2` are in the same pull request. So the leftover of `jack2` build take effect on building `mpv`? That case should we keep `TERMUX_PKG_BUILD_DEPENDS="jack2"`?

Or should I remove `TERMUX_PKG_ANTI_BUILD_DEPENDS` which I copied from `vulkan-loader`? Is that `vulkan-loader` uses anti build deps because the that two libs can't be linked in the same way? while jack2 and pipewire-jack can be linked the same way.